frontend/packages/topology/src/elements/BaseGraph.ts

```diff
@@ -25,6 +25,8 @@ export default class BaseGraph<E extends GraphModel = GraphModel, D = any> exten
 
   private currentLayout?: Layout;
 
+  private zoomExtents: [number, number] = [0.25, 4];
```

Comment:
I wonder if we want to type the `[number, number]`... you use it frequently. Would it make sense to type it?